src/gentropy/finngen_finemapping_ingestion.py

```diff
@@ -37,6 +37,10 @@ def __init__(
             finngen_susie_finemapping_cs_summary_files=finngen_susie_finemapping_cs_summary_files,
         )
 
+        finngen_finemapping_df = finngen_finemapping_df.validate_lead_pvalue(
```

Comment:
Minor stylistic comment, and it is absolutely my preference, but I like to use as few variables as possible:
```python
(
    # Reading Finngen finemapped dataset and convert it to study locus:
    FinnGenFinemapping.from_finngen_susie_finemapping(
        spark=session.spark,
        finngen_susie_finemapping_snp_files=finngen_susie_finemapping_snp_files,
        finngen_susie_finemapping_cs_summary_files=finngen_susie_finemapping_cs_summary_files,
    )
    # Flagging sub-significnat loci:
    .validate_lead_pvalue(
        pvalue_cutoff=FinngenFinemappingConfig().finngen_finemapping_lead_pvalue_threshold
    )
    # Write the output.
    .df.write.mode(session.write_mode).parquet(
        finngen_finemapping_out
    )
)
```

Reply:
Please carefully check, I removed the variables but not sure